app-creator/src/widgets/attributes-tab/attributes-tab.ts

```diff
@@ -391,10 +397,10 @@ ${value}</textarea
     });
   }
 
-  getUniqueAttributes() {
+  getUniqueAttributes(): (TemplateResult | {})[] | {} {
```

Comment:
Prefer `Array<TemplateRestult|{}>` here for the return type, since it's more complex. See [go/tsstyle#arrayt-type](go/tsstyle#arrayt-type). Similar for the function below

Reply:
link is blocked :/